setup.cfg

```diff
@@ -1,5 +1,62 @@
 [metadata]
-description-file = README.md
+url = https://github.com/nipype/pydra
+author = nipype developers 
+author_email = neuroimaging@python.org
+maintainer = nipype developers
+maintainer_email = neuroimaging@python.org
+description = Pydra dataflow engine
+long_description = file:long_description.rst
+long_description_content_type = text/x-rst; charset=UTF-8
+license = Apache License, 2.0
+provides =
+    pydra
+classifiers =
+    Development Status :: 2 - Pre-Alpha
+    Environment :: Console
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: Apache Software License
+    Operating System :: MacOS :: MacOS X
+    Operating System :: POSIX :: Linux
+    Programming Language :: Python :: 3.7
+    Topic :: Scientific/Engineering
+
+[options]
+python_requires = >= 3.7
+install_requires =
+    pytest >= 4.4.0
```

Comment:
@satra @djarecka It's not clear that this is actually a dependency, as pytest isn't imported outside of `test_*.py` files. Should I move it to the extras_require?

Reply:
i think we can move it to extra_require